NavigationSystem/ASPire.mk

```diff
@@ -0,0 +1,59 @@
+###############################################################################
+#
+# ASPire Makefile
+#
+###############################################################################
+
+
+###############################################################################
+# Files
+###############################################################################
+
+MAIN = main_ASPire.cpp
+OBJECTS = $(addprefix $(BUILD_DIR)/, $(SRC:.cpp=.o))
```

Comment:
Place the object file creation at the end of the Files part. First all the SRC files have to be defined.